.github/workflows/build.yml

```diff
@@ -15,11 +15,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        rails: ["~> 7.0.0", "~> 6.1.0", "~> 6.0.0"]
+        rails: ["~> 7.1.0", "~> 7.0.0", "~> 6.1.0", "~> 6.0.0"]
         ruby: ["3.2.2", "3.1.4", "3.0.6", "2.7.8"]
         include:
           - ruby: 3.2
             rails: 'edge'
+          - ruby: 3.2
+            rails: '~> 7.1.0'
```

Comment:
That's duplicate run from the above matrix setting.

Reply:
This might not be redundant, as it ensures that you are testing against the latest 3.2.x version of Ruby.

Reply:
I don't think we need to test both 3.2.2 and the future 3.2.3, I rather update to 3.2.3 for every rails version when it comes out